src/test/ui/pattern/bindings-after-at/borrowck-pat-at-and-box.rs

```diff
@@ -18,53 +18,51 @@ fn nc() -> NC {
 
 fn main() {
     let a @ box &b = Box::new(&C);
-    //~^ ERROR use of moved value
 
     let a @ box b = Box::new(C);
-    //~^ ERROR use of moved value
 
     fn f1(a @ box &b: Box<&C>) {}
-    //~^ ERROR use of moved value
 
     fn f2(a @ box b: Box<C>) {}
-    //~^ ERROR use of moved value
 
     match Box::new(C) {
-        a @ box b => {} //~ ERROR use of moved value
```

Comment:
These patterns are all of a sudden legal now.

Reply:
These are expected. The inner thing is now first copied and then the outer thing is moved. `C` is `Copy`, so this works out.